codecov.yml

```diff
@@ -11,8 +11,20 @@ ignore:
 beta_groups:
   - "labels"
 
+services:
+  minio:
+    hash_key: 12345bf3f7d947f2a0681b2154067890
+    verify_ssl: false
+    host: "minio"
+    port: 9000
+    # bucket: <bucket-name>
+    # region: <bucket-region>
+    access_key_id: codecov-default-key
+    secret_access_key: codecov-default-secret
+    client_uploads: true
+
```

Comment:
i know there is some mock config stuff set up in https://github.com/codecov/shared/blob/main/tests/conftest.py but i don't actually know if/where there is a yml for the docker-compose setup. it may be that nobody needed one. if we need one now, put it in the `docker` folder?

Reply:
`hash_key` is fake, I manually used prepend/append 12345 67890 to make it fake

Its late so not fully following the rest of the ask here, should I move this to a docker folder?

Reply:
i think this `codecov.yml` is the one that defines our flags and stuff for our CI uploads, not the `codecov.yml` that defines the service config inside docker-compose. i don't think this will be read by anything. if you just delete it, does everything still work?